GiniSwitchSDK/GiniSwitchSDK/Classes/OnboardingPageViewController.swift

```diff
@@ -10,8 +10,8 @@ import UIKit
 
 class OnboardingPageViewController: UIViewController {
```

Comment:
You missed this one.

Reply:
I thought about and but decided not to make it final. It's a long shot, but theoretically, if someone what's to have a different kind of onboarding page, they might decide to subclass this one.